src/gauge.js

```diff
@@ -131,11 +131,38 @@
       return percentage * gaugeSpanAngle / 100;
     }
 
-    function normalize(value, min, limit) {
-      var val = Number(value);
-      if(val > limit) return limit;
-      if(val < min) return min;
-      return val;
+    /**
+     * Check value. If value is number - set it to array
+     * @param {Array/Number} value
+     * @param {Number} min
+     * @param {Number} limit
+     * @param {String} color
+     * @return {Array}
+     */
+    function normalizeMultipleValue(value, min, limit, color) {
+      // If value is single (number) - set it ro array
+      // Value should be in format [{value: number, color: 'string'}]
+      if (!Array.isArray(value)) value = [{value, color}];
+
+      var totalValue = 0;
+      var result = [];
+
+      value.forEach(function (v) {
```

Comment:
do we have any updates here? logic is not the same as it was before

Reply:
This new  logic for multiple data